src/main/java/org/jitsi/videobridge/ratecontrol/BitrateController.java

```diff
@@ -210,7 +214,7 @@ public int calcNumEndpointsThatFitIn()
         long remainingBandwidth = availableBandwidth;
         int numEndpointsThatFitIn = 0;
 
-        final Iterator<Endpoint> it = channel.getReceivingEndpoints();
+        final Iterator<Endpoint> it = null; //channel.getReceivingEndpoints();
```

Comment:
Is the BitrateController actually used? If it is this would cause an NPE in a few in the while statement.